src/serde_arrow_buffer.erl

```diff
@@ -53,6 +53,44 @@ new(Values, Type) ->
     Len = byte_size(Bin),
     from_binary(Bin, Type, Len).
 
+%% @doc Creates a new buffer from a list of Erlang values or binaries, given its
+%% type
+%% @end
+-spec from_erlang(
+    Value :: [serde_arrow_type:native_type()],
+    Type :: serde_arrow_type:arrow_longhand_type()
+) ->
+    Buffer :: #buffer{}.
+from_erlang(Data, Type) ->
+    Len =
+        case Type of
+            {bin, undefined} ->
+                lists:foldl(
+                    fun
+                        (Bin, Len) when Bin =:= undefined; Bin =:= nil ->
+                            1 + Len;
+                        (Bin, Len) ->
+                            byte_size(Bin) + Len
+                    end,
+                    0,
+                    Data
+                );
+            _ ->
+                length(Data) * serde_arrow_type:byte_length(Type)
+        end,
+    #buffer{type = Type, length = Len, data = Data}.
+
+%% @doc God knows
```

Comment:
:D

Reply:
I forgot to commit the docs. That was simply some placeholder text to remind me to write it up 😅